<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dev_tool</class>
 <widget class="QMainWindow" name="Dev_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dev_tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>溢出数恢复</string>
     </attribute>
     <widget class="QLineEdit" name="num_input">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>溢出数：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>计算结果</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8位结果</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>16位结果</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>380</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>32位结果</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>370</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>64位结果</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="bit8_ret">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="bit16_ret">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="bit32_ret">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>420</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="bit64_ret">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>420</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="calc_num_btn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>计  算</string>
      </property>
     </widget>
     <widget class="QLabel" name="status_label_tab1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string> 初始化中......</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>NV_OMAP生成</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>UTC时间戳转换</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Made by Serenity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
